backend/src/routes/v2/schema/getSchemas.ts

```diff
@@ -24,7 +24,7 @@ export const getSchemas = [
   async (req: Request, res: Response<GetSchemaResponse>) => {
     const { query } = parse(req, getSchemasSchema)
 
-    const schemas = await findSchemasByKind(query.kind)
+    const schemas = await getSchemasByKind(query.kind)
```

Comment:
Am tempted to say that request functions should have names like `getSchemas` and `postResult`, etc.  Services should be independent of HTTP request language and so use phrasing like `createSchema` or `findSchemas` or `readSchema`.

Reply:
I made this change to follow the convention you've implemented in your PR....